src/test/java/seedu/modtracker/ParserTest.java

```diff
@@ -0,0 +1,66 @@
+package seedu.modtracker;
+
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    ModuleList modList = new ModuleList();
+    Parser parser = new Parser();
+
+    @Test
+    public void parse_invalidInput_expectPrintInvalid() {
+        ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outContent));
+
+        String input = "exit addmod ";
+        parser.parse(input, modList, null, null, true);
+        String expected = "OOPS!!! I'm sorry, but I don't know what that means :-(" + System.lineSeparator();
```

Comment:
A suggestion is that this string can be set as a constant in the UI class and then you can call the constant (UI.constant name) here.  If the message is changed at the other class for whatever reason then this part will automatically change also so don't really need change manually every time.